packages/core/src/zip/ImmutableZipper.js

```diff
@@ -0,0 +1,37 @@
+import { makeZipper } from 'zippa';
+import { Iterable, List } from 'immutable'
+
+class ImmutableZipper {
+
+  childrenElements
+  zipper
+
+  isBranch(element) {
+    return !!element.get(this.childrenElements)
```

Comment:
There can be multiple children properties (think split-pane container). The zipper does the following:
- from an element node  
- z.down from an element *always* takes you to a `childrenCollection` node  (one per named children property)
- z.down from the childrenCollection node takes you again to an element

e.g.
```javascript
const data =  {
  kind: 'split-pane',
  left: [ /* left pane elements */ ],
  right: [ /* right pane elements */ ],
};

const root = zip.elementZipper(data);
const leftChildren = z.down(root);
const rightChildren = z.right(leftChildren);
```